spec/unit/outputs/opensearch/http_client_spec.rb

```diff
@@ -162,6 +162,19 @@
     end
   end
 
+  describe "legacy_template" do
```

Comment:
Similar to my other testing comment. It might be better to test and verify the behavior of `template_install` instead. This is the method which is called by template manager.

Reply:
updated the test to invoke `template_install`